client/css/style.css

```diff
@@ -2708,6 +2709,7 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 
 /* Image viewer and drag-and-drop overlay */
 
+#confirm-dialog-overlay,
```

Comment:
I'm beginning to feel we might want to have a generic overlay class for all of these overlays to use. Maybe that can be done in another PR if we think it's worth it.

Reply:
Yeah probably should figure something out, but there's slight differences between all of them, so that doesn't help.